### PR TITLE
Update system-requirements.md to mention same number of nodes at each site

### PR DESCRIPTION
**Pull request by cpuck**
Added "Each site must have the same number of nodes and drives." to the first bullet point under server requirements.

### CODE REVIEW

**Comment by PRMerger5** [comment]
@cpuck : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by ManikaDhiman** [comment]
#sign-off

**Comment by PRMerger5** [comment]
@ManikaDhiman: I'm sorry - only the author of this article, @jasongerend, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: azure-stack/hci/concepts/system-requirements.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ A standard Azure Stack HCI cluster requires a minimum of two servers and a maxim
 
 Keep the following in mind for various types of Azure Stack HCI deployments:
 
-- Stretched clusters require servers to be deployed at two separate sites. The sites can be in different countries, different cities, different floors, or different rooms. A stretched cluster requires a minimum of 4 servers (2 per site) and a maximum of 16 servers (8 per site).
+- Stretched clusters require servers to be deployed at two separate sites. The sites can be in different countries, different cities, different floors, or different rooms. A stretched cluster requires a minimum of 4 servers (2 per site) and a maximum of 16 servers (8 per site). Each site must have the same number of nodes and drives.
```
Comment (JasonGerend) [inline azure-stack/hci/concepts/system-requirements.md:24]:
```suggestion
- Stretched clusters require servers to be deployed at two separate sites. The sites can be in different countries, different cities, different floors, or different rooms. A stretched cluster requires a minimum of 4 servers (2 per site) and a maximum of 16 servers (8 per site). Each site must have the same number of servers and drives.
```

Comment (JasonGerend) [reply azure-stack/hci/concepts/system-requirements.md:24]: We haven't introduced the "nodes" term yet at this point, so we should probably stick with "servers" right here.

**Review by JasonGerend — CHANGES_REQUESTED** [review]
We haven't introduced the "nodes" term yet at this point, so we should probably stick with "servers" right here.

**Comment by ManikaDhiman** [comment]
#please-close

**Pull request closed without merging**